.github/workflows/cf-deploy-react.yml

```diff
@@ -10,6 +10,7 @@ on:
       - main
     paths:
       - 'packages/react/**'
+      - 'examples/react-widget-app/**'
```

Comment:
lets remove paths filters, we want new deployment even if widget package is modified^^